README.md

```diff
@@ -597,6 +597,9 @@ config :fun_with_flags, :cache_bust_notifications,
   enabled: true,
   adapter: FunWithFlags.Notifications.PhoenixPubSub,
   client: :my_pubsub_process_name
+  # pubsub_driver: Phoenix.PubSub.PG2 (optional)
+  # pubsub_driver: {Phoenix.PubSub.PG2, pool_size: 10} (optional with options)
```

Comment:
I think it would be clearer if the commented-out code was valid Elixir. That mean that the text in parentheses should be its own comment.
```suggestion
  # pubsub_driver: Phoenix.PubSub.PG2                  # optional
  # pubsub_driver: {Phoenix.PubSub.PG2, pool_size: 10} # optional with options
```